addons/im_livechat/static/src/embed/common/chatbot/chatbot_model.js

```diff
@@ -138,6 +151,11 @@ export class Chatbot extends Record {
         this.currentStep.completed = stepCompleted;
     }
 
+    async _processMultiLineAnswer(message) {
+        this.tmpAnswer = this.thread.composer.text;
+        return Promise.resolve().then(() => this._processAnswerDebounced(message));
+    }
```

Comment:
function is `async`, so can put `await Promise.resolve()` on its own line.
This is most likely motivated to make it work with debounce, as this function is called inside debounce this is to ensure the call again is properly debounced.
Putting `await Promise.resolve()` on its own line allows to put an inline comment to justify it.